vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java

```diff
@@ -319,29 +403,48 @@ private XLarge() {
              */
             public static final class XXLarge {
 
+                public static final String BLOCK = "2xl:block";
                 public static final String FLEX = "2xl:flex";
+                public static final String GRID = "2xl:grid";
                 public static final String HIDDEN = "2xl:hidden";
+                public static final String INLINE = "2xl:inline";
+                public static final String INLINE_BLOCK = "2xl:inline-block";
+                public static final String INLINE_FLEX = "2xl:inline-flex";
+                public static final String INLINE_GRID = "2xl:inline-grid";
 
                 private XXLarge() {
                 }
             }
         }
     }
 
+    /**
+     * Classes for setting borders between elements.
+     */
+    public static final class Divide {
+        public static final String X = "divide-x";
+        public static final String Y = "divide-y";
+
+        private Divide() {
+
+        }
+    }
+
     /**
      * Classes for setting how items grow and shrink in a flexbox layout.
      * Applies to flexbox items.
      */
     public static final class Flex {
 
+        public static final String ONE = "flex-1";
         public static final String AUTO = "flex-auto";
         public static final String NONE = "flex-none";
 
-        public static final String GROW = "flex-grow";
-        public static final String GROW_NONE = "flex-grow-0";
+        public static final String GROW = "grow";
+        public static final String GROW_NONE = "grow-0";
 
-        public static final String SHRINK = "flex-shrink";
-        public static final String SHRINK_NONE = "flex-shrink-0";
+        public static final String SHRINK = "shrink";
+        public static final String SHRINK_NONE = "shrink-0";
```

Comment:
I don't see these changes in the web components repo. Also, it would be better to extract renaming classes into separate PR (as it's changing the existing classes, not related to adding missing classes).

Reply:
UPD: web components classes renaming is done in https://github.com/vaadin/web-components/pull/7515. I still think we should extract change from `flex-grow` to `grow` etc into separate PR at least to have a separate changelog entry.

Reply:
I'll make a separate PR.